lib/elixir/test/elixir/macro_test.exs

```diff
@@ -652,36 +652,6 @@ defmodule MacroTest do
              """
     end
 
-    test "unless expression" do
```

Comment:
I'd go ahead and remove the dbg feature altogether.

Reply:
The one specific to unless.

Reply:
Agreed!

Reply:
[afd5d11](https://github.com/elixir-lang/elixir/pull/13850/commits/afd5d114cb0d6a328675f8287d701fe9b23cafad)